### PR TITLE
Add user test

### PR DESCRIPTION
**Pull request by jarindr**
Issue/Task Number: 508-refactoring-everything-and-add-tests

# Overview
Add test for users and current user.

# Changes
- Add test for users and current user action.
- Add test for users and current reducer.
- Renaming.
- Fix minor typo bugs.

# Usage
run `yarn test`

# Impact
deploy as usual

### CODE REVIEW

File: apps/admin_panel/assets/src/omg-user-current/currentUserProvider.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ class UserProvider extends Component {
   componentDidMount = async () => {
     if (this.props.currentUserLoadingStatus === 'DEFAULT') {
       const result = await this.props.getCurrentUser()
-      if (!result.data.success) {
+      if (!result.data) {
```
Comment (T-Dnzt) [inline apps/admin_panel/assets/src/omg-user-current/currentUserProvider.js]: Why the change? Shouldn't you be checking the error code as well? Or maybe it's displayed after the push?

File: apps/admin_panel/assets/src/omg-user-current/action.js [hunk 2]
```diff
@@ -1,51 +1,27 @@
...
-  } catch (error) {
-    return dispatch({ type: 'CURRENT_USER/REQUEST/FAILED', error })
-  }
-}
+import { createActionCreator } from '../utils/createActionCreator'
+export const getCurrentUser = () =>
+  createActionCreator({
+    actionName: 'CURRENT_USER',
+    action: 'REQUEST',
+    service: currentUserSerivce.getCurrentUser
```
Comment (T-Dnzt) [inline apps/admin_panel/assets/src/omg-user-current/action.js]: service, not serivce :)

File: apps/admin_panel/assets/src/omg-user-current/currentUserProvider.js (hunk 1 above)
Comment (jarindr) [reply apps/admin_panel/assets/src/omg-user-current/currentUserProvider.js]: This is a data provider component, we don't need to handle any error code here.
if the returned result has data, it is guaranteed to be a success action. if it is error, it will return `result.error`
